change/@ni-spright-angular-508c5fd4-c76d-4e41-becf-8c5a1dd724e2.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Added @atmgrifter00 as a reviewer since presumably we will use the same owners for spright-angular as nimble-angular when we get around to configuring CODEOWNERS (I think that's tracked in a follow up task).